features/promotion/managing_catalog_promotions/editing_catalog_promotion.feature

```diff
@@ -44,13 +44,13 @@ Feature: Editing catalog promotion
         When I want to modify a catalog promotion "Christmas sale"
         Then I should not be able to edit its code
 
-    @api
+    @api @ui @javascript
     Scenario: Editing catalog promotion rule
         When I want "Christmas sale" catalog promotion to be applied on "Kotlin T-shirt" variant
         Then I should be notified that it has been successfully edited
         And this catalog promotion should be applied on "Kotlin T-shirt" variant
```

Comment:
What about a "PHP T-Shirt" variant? Is this promotion still applying to this variant? I would be for adding a step to check that

Reply:
Added another step to check that